src/common/plugin/types.ts

```diff
@@ -161,21 +161,22 @@ export type FeeInfo = FeeRates & {
   standardFeeHighAmount: string
   maximumFeeRate?: string
 }
-export const asFeeInfo = asObject<FeeInfo>({
-  ...asFeeRates.shape,
-
-  lowFeeFudgeFactor: asMaybe(asString),
-  standardFeeLowFudgeFactor: asMaybe(asString),
-  standardFeeHighFudgeFactor: asMaybe(asString),
-  highFeeFudgeFactor: asMaybe(asString),
-
-  // The amount of satoshis which will be charged the standardFeeLow
-  standardFeeLowAmount: asString,
-  // The amount of satoshis which will be charged the standardFeeHigh
-  standardFeeHighAmount: asString,
-  // A safe-guard for any potential software bugs:
-  maximumFeeRate: asOptional(asString)
-})
+export const asFeeInfo = (fallback?: FeeInfo): Cleaner<FeeInfo> =>
+  asObject<FeeInfo>({
+    ...asFeeRates.shape,
+
+    lowFeeFudgeFactor: asMaybe(asString),
+    standardFeeLowFudgeFactor: asMaybe(asString),
+    standardFeeHighFudgeFactor: asMaybe(asString),
+    highFeeFudgeFactor: asMaybe(asString),
+
+    // The amount of satoshis which will be charged the standardFeeLow
+    standardFeeLowAmount: asString,
+    // The amount of satoshis which will be charged the standardFeeHigh
+    standardFeeHighAmount: asString,
+    // A safe-guard for any potential software bugs:
+    maximumFeeRate: asOptional(asString, fallback?.maximumFeeRate)
```

Comment:
Even though it's not really used yet, it seems like the fallback should be added to all the `asMaybe` parameters otherwise the `fallback` param is misleading as it really only applies to the `maximumFeeRate`